Resources/config/default_settings.yml

```diff
@@ -13,3 +13,26 @@ parameters:
     # Type ('js' or 'template')
     # ez_platformui.<scope>.yui.modules.<module_name>.type
     ez_platformui.default.css.files: []
+
+    ezsettings.default.system_info_view:
```

Comment:
`default` didn't work out for me, had to change it to `global`.

For some reason, `ezsettings.site.system_info_view` was undefined, while it was set for default. I guess that PlatformUI was running in that siteaccess for me.